File: common/persons/src/main/java/ru/tensor/sbis/persons/recipientselection/RecipientSelectionItem.java
package ru.tensor.sbis.persons.recipientselection;

import android.os.Parcelable;

import java.util.UUID;

/**
 * Created by se.petrova on 12/1/17.
 */

public interface RecipientSelectionItem extends Parcelable {
    /**
     * @return amount of contacts at recipient item
     */
    int getItemCount();

    /**
     * @return uuid
     */
    UUID getUUID();
}

File: declaration/communication-decl/src/main/java/ru/tensor/sbis/communication_decl/selection/recipient/data/RecipientSelectionItem.kt
package ru.tensor.sbis.communication_decl.selection.recipient.data

import java.util.UUID

/**
 * Интерфейс элемента выбора получателей.
 *
 * @author vv.chekurda
 */
interface RecipientSelectionItem {

    /**
     * Идентификатор элемента.
     */
    val uuid: UUID

    /**
     * Заголовок элемента.
     */
    val title: String
}